src/index.js

```diff
@@ -92,6 +92,10 @@ function radredis(schema, transforms, port, host, options){
         .map((objs) => { this.push(objs) })
         .then(() => callback() )
       }))
+    },
+
+    count: () => {
+      return redis.get(`${keyspace}:id`).then(count => parseInt(count, 10))
```

Comment:
Issue with doing this is deletions. Ids are incremented and lower ids can be removed. I think you should count the `id` index directly